### ISSUE TITLE
Headers file is invalid even though it should be valid?

### ISSUE THREAD

**Issue by Alphare** [ccgauche/ytermusic#48]
During troubleshooting for #47, I re-created my `headers.txt` file and it seems to never be valid even though I did everything the same as before. I've tried multiple times, not sure if anything has changed.

**Comment by zopieux** [comment]
Yeah, I was affected as well. It's not documented in the README, but you also need to pass a good-looking User-Agent header in `headers.txt` (e.g. a recent Chrome), otherwise YouTube serves an error page when ytermusic expects some API tokens.

```
User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36
Cookie: …
```

works here.

**Comment by zopieux** [comment]
@ccgauche Some suggestions here:

1. validate that the user provides a User-Agent and have a more explicit error if not
1. consider providing a default user-agent – but this will require some permanent maintenance work as Google will decide it's not good at some point
1. document the need for User-Agent in the readme setup
1. https://github.com/ccgauche/ytermusic/blob/9c076786d5ead15b1c4607c0f57f7040999b096a/src/main.rs#L71-L73 does a different validation than what https://github.com/ccgauche/ytermusic/blob/9c076786d5ead15b1c4607c0f57f7040999b096a/ytpapi2/src/lib.rs#L36-L37 expects, that is, you validate for any case in `main`, but then you only accept Kebab case in `lib`. I'd suggest matching on the lowercase'd name in `lib.rs`.

**Comment by ccgauche** [comment]
Thanks for the feedback! Improvements on troubleshooting are planned and some hot-fixes will be released tomorrow.

**Comment by ccgauche** [comment]
Fixed by 2f2eebd

**Comment by ndom91** [comment]
Adds "User Agent" config option to README: https://github.com/ccgauche/ytermusic/pull/50
